.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Update gitignore file and use this template https://github.com/github/gitignore/blob/master/Node.gitignore

Reply:
done